app/src/test/java/com/nervousfish/nervousfish/ExampleUnitTest.java

```diff
@@ -13,17 +21,27 @@
  */
 @SuppressWarnings("PMD")
 public class ExampleUnitTest {
+    ISample sample = mock(ISample.class);
+
     @Test
     public void addition_isCorrect() throws Exception {
```

Comment:
It was there by default, but it doesn't matter as this class will be removed once the first real unittests are added to the repo